java.js

```diff
@@ -13,3 +13,144 @@ Items.forEach((n) => {
     menu.classList.toggle('hide');
   });
 });
+const projectDetails = [{
+  name: 'Keeping track of hundreds  of components website',
+  name2: 'Multi Post Stories',
+  detail: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  detail2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  image: './img/Snapshoot.png',
+  technologies: [
+    'HTML/CSS',
+    'Ruby on Rails',
+    'JavaScript',
+  ],
+  live: 'https://github.com/Uchenna-ibute/Portfolio',
+  source: 'https://github.com/Uchenna-ibute',
+},
+{
+  name: 'Keeping track of hundreds  of components website',
+  name2: 'Multi Post Stories',
+  detail: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  detail2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  image: './img/Snapshoot.png',
+  technologies: [
+    'HTML/CSS',
+    'Ruby on Rails',
+    'JavaScript',
+  ],
+  live: 'https://github.com/Uchenna-ibute/Portfolio',
+  source: 'https://github.com/Uchenna-ibute',
+},
+{
+  name: 'Keeping track of hundreds  of components website',
+  name2: 'Multi Post Stories',
```

Comment:
- [ OPTIONAL ] Good job adding data for each card dynamically as requested by the previous reviewer, I suggest writing at least a different title or name in your case to make sure the user sees the difference :thinking: